csfieldguide/templates/interactives/matrix-simplifier.html

```diff
@@ -106,6 +106,22 @@ <h5 class="col-12 text-center">{% trans "Result" %}</h5>
       </div>
     </div>
 
+    <div class="row justify-content-center mb-1">
+      <small class="text-muted text-center">{% trans "The format below can be copied for pasting into the scene editor interactive." %}</small>
+    </div>
+
+    <div class="row justify-content-center mb-2">
+      <textarea id="code-to-copy" class="form-control col-md-4 col-sm-12" readonly></textarea>
+    </div>
+
+    <div class="row justify-content-center mb-5">
+      <button id="copy-eqtn" type="button" class="btn btn-primary col-md-2 col-sm-6 d-none" data-toggle="tooltip" data-placement="bottom">{% trans "Copy equation" %}</button>
```

Comment:
There is no tooltip for before the button is clicked. the RSA interactive equivalents have an extra `title` part for this